aes/ta/aes_ta.c

```diff
@@ -247,8 +247,8 @@ static TEE_Result set_aes_key(void *session, uint32_t param_types,
 	key_sz = params[0].memref.size;
 
 	if (key_sz != sess->key_size) {
-		EMSG("Worng key size %d, expect %d bytes",
-				key_sz, sess->key_size);
+		EMSG("Wrong key size %d, expect %d bytes",
```

Comment:
`%d` is okay for an unsigned ?

Reply:
`%lu`? `%zu`?

Reply:
Or even the `PRIu32`?

Reply:
Given that `key_sz` and `sess->key_size` is changed to a `uint32_t` you should use `% PRIu32`. Before with `size_t` `%zu` would a have been appropriate.